### PR TITLE
Fix Grammar Issue

### PR DESCRIPTION
**Pull request by krishdevdb**
closes #ISSUE-NO

#### What does this PR do?
Fixes Grammar in issue template of gh-profile-review

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/gh-profile-review.md [hunk 1]
```diff
@@ -11,4 +11,4 @@ assignees: ''
 
 You can also join the discord community [here](http://discord.eddiehub.org)
 
-Feel free to check out other cool repositories of EddieHub Community [here](https://github.com/EddieHubCommunity)
+Feel free to check out other cool repositories of The EddieHub Community [here](https://github.com/EddieHubCommunity)
```
Comment (schmelto) [inline .github/ISSUE_TEMPLATE/gh-profile-review.md:14]:
```suggestion
Feel free to check out other cool repositories of the EddieHub Community [here](https://github.com/EddieHubCommunity)
```
maybe a small `t`

Comment (krishdevdb) [reply .github/ISSUE_TEMPLATE/gh-profile-review.md]: Thanks for the suggestion. Just made it

File: .github/ISSUE_TEMPLATE/opensource-review.md [hunk 2]
```diff
@@ -11,4 +11,4 @@ assignees: ''
 
 You can also join the discord community [here](http://discord.eddiehub.org)
 
-Feel free to check out other cool repositories of EddieHub Community [here](https://github.com/EddieHubCommunity)
+Feel free to check out other cool repositories of The EddieHub Community [here](https://github.com/EddieHubCommunity)
```
Comment (schmelto) [inline .github/ISSUE_TEMPLATE/opensource-review.md:14]:
```suggestion
Feel free to check out other cool repositories of the EddieHub Community [here](https://github.com/EddieHubCommunity)
```

File: .github/ISSUE_TEMPLATE/portfolio-review.md [hunk 3]
```diff
@@ -12,4 +12,4 @@ Portfolio link:
 
 You can also join the discord community [here](http://discord.eddiehub.org)
 
-Feel free to check out other cool repositories of EddieHub Community [here](https://github.com/EddieHubCommunity)
+Feel free to check out other cool repositories of The EddieHub Community [here](https://github.com/EddieHubCommunity)
```
Comment (schmelto) [inline .github/ISSUE_TEMPLATE/portfolio-review.md:15]:
```suggestion
Feel free to check out other cool repositories of the EddieHub Community [here](https://github.com/EddieHubCommunity)
```

File: .github/ISSUE_TEMPLATE/resume-review.md [hunk 4]
```diff
@@ -12,4 +12,4 @@ Resume link:
 
 You can also join the discord community [here](http://discord.eddiehub.org)
 
-Feel free to check out other cool repositories of EddieHub Community [here](https://github.com/EddieHubCommunity)
+Feel free to check out other cool repositories of The EddieHub Community [here](https://github.com/EddieHubCommunity)
```
Comment (schmelto) [inline .github/ISSUE_TEMPLATE/resume-review.md:15]:
```suggestion
Feel free to check out other cool repositories of the EddieHub Community [here](https://github.com/EddieHubCommunity)
```

**Review by schmelto — APPROVED** [review]

**Review by kaiwalyakoparkar — APPROVED** [review]
LGTM :+1:

**Review by vinzvinci — APPROVED** [review]
LGTM 👍

**Review by bishalr0y — COMMENTED** [review]
LGTM🚀

**Review by adityaraute — APPROVED** [review]
Looks Good. Should be merged soon.

**Review by kaiwalyakoparkar — APPROVED** [review]
LGTM :rocket:

**Review by Vyvy-vi — APPROVED** [review]
Looks good to me!
*I'm not entirely sure if "the" can be omitted here, but this PRs good :)

**Pull request merged**